app/views/teachers/classrooms/students/edit_quiz_score.html.slim

```diff
@@ -0,0 +1,19 @@
+.row
+  .small-12.columns
+    .center.paper
+      .row
+        .small-12.columns
+          h2 = t ".heading", student_email: @student.email
+
+          .spacer-small
+
+          = render "form_errors", obj: @student
+          = simple_form_for @student,
+            url: edit_quiz_score_student_path(@student, @quiz) do |f|
+            .row
+                .medium-8.medium-centered.columns
+                = f.input @quiz.student_column_name
```

Comment:
nit: You didn't nest this input under the div. It should be indented one more level to actually make it be inside the `.medium-8.medium-centered.columns` div.